fastlane/lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -10,7 +10,16 @@ def self.run(params)
         begin
           path = File.expand_path(params[:path])
           plist = Plist.parse_xml(path)
-          plist[params[:key]] = params[:value]
+          if params[:subkey]
+            if !plist[params[:key]]
```

Comment:
well yes, this is about my lack of ruby knowledge, sorry. Anyway I have rewritten it for positive test first, so no `unless` is needed here.